oak/server/oak_node.cc

```diff
@@ -309,7 +318,9 @@ grpc::Status OakNode::ProcessModuleInvocation(grpc::GenericServerContext* contex
   LOG(INFO) << "Created gRPC method name channel";
 
   // Create the logging channel, used by the module to log statements for debugging.
-  std::unique_ptr<Channel> logging_channel = absl::make_unique<LoggingChannel>();
+  // Each log message gets a prefix holding the name of the gRPC method being
```

Comment:
Why though? It seems quite specific, and I don't see a good reason for the Oak VM to handle this; if the Oak Module wants to log in a particular way, it can already do that itself.

Reply:
The aim was just to get have a bit more auto-generated context available for log messages, because the current emitted context is entirely irrelevant (it's always `INFO  logging_channel.h : 39 : LOG:`). 

Having a logging library that modules could link in (#96) would solve the problem, so I can just drop the final commit in the expectation of that coming along.